pkgs/applications/networking/cluster/kns/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenvNoCC, lib, fetchFromGitHub, bash }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "kns";
+  version = "50b6370c88136a580599c913f0fae2cb24981dc1";
```

Comment:
Versions without specific upstream releases should be named like `unstable-YYYY-MM-DD`. 

https://nixos.org/manual/nixpkgs/unstable/#sec-package-naming

Reply:
resolved in: https://github.com/NixOS/nixpkgs/pull/193995/commits/af41e1d01871ff11c396e1d74be126b508453f15